app/js/components/tag.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+import React, { Component } from 'react';
+
+export default class Tag extends Component {
+    render() {
+      return(
+            <button className='qdb-tag-button btn btn-primary btn-sm' id='tag-{this.props.tag}'>{this.props.tag}</button>
```

Comment:
I don't think that syntax for `id=` would work...

Make sure the id is working. I think you'd need ``id={`tag-${this.props.tag}`}``